gateleen-validation/src/test/java/org/swisspush/gateleen/validation/ValidatorTest.java

```diff
@@ -61,20 +69,12 @@ public void testValidationWithVariables(TestContext context){
                 "    \"type\": \"object\"\n" +
                 "}");
 
-        class GETValidationResourceRequest extends HttpServerRequestMock {
-            @Override public HttpMethod method() {
-                return HttpMethod.GET;
-            }
-            @Override public String uri() { return "/foo/mediadata/v1/specials/03";   }
-            @Override public String path() {
-                return "/foo/mediadata/v1/specials/03";
-            }
-        }
+        CustomHttpServerRequest gETValidationResourceRequest = new CustomHttpServerRequest(HttpMethod.GET, "/foo/mediadata/v1/specials/03");
 
         String type = "GET/out";
         Buffer jsonBuffer = Buffer.buffer();
         final SettableFuture<String> future = SettableFuture.create();
-        validator.validate(new GETValidationResourceRequest(), type, jsonBuffer, validationResult -> {
+        validator.validate(gETValidationResourceRequest, type, jsonBuffer, validationResult -> {
```

Comment:
```java
gETValidationResourceRequest
^^^^--- Nothing wrong. uPpeR/loWeR case just looks funny (aka counter-intuitive) :)
```
Maybe the classname should start with `GetVa...` instead `GETVa...`? (AFAIK `GET` is not an acronym)